code/datums/uplink_item.dm

```diff
@@ -157,53 +157,53 @@ var/list/uplink_items = list()
 	name = "Spirit Camera"
 	desc = "This special camera allows you to take long-distance photographs of any living person provided you know their name."
 	item = /obj/item/device/camera/spiritcam
-	cost = 4
+	cost = 1
 	jobs = list("Librarian", "Assistant", "Clown", "Mime", "Lawyer", "Chaplain")
 
 //LIBRARIAN
 /datum/uplink_item/job_specific/soulstone
 	name = "Soulstone"
 	desc = "This stone will be able to capture your victim's soul and bind them to your will."
 	item = /obj/item/device/soulstone/anybody
-	cost = 5 //nerfed the cost on Chronitonity's request
+	cost = 4 //nerfed the cost on Chronitonity's request
 	jobs = list("Librarian")
 
 //CHAPLAIN
 /datum/uplink_item/job_specific/skelestone
 	name = "Skelestone"
 	desc = "Make a skeleton minion! Has one use."
 	item = /obj/item/device/necromantic_stone/oneuse
-	cost = 7
+	cost = 6
 	jobs = list("Chaplain")
 
 /datum/uplink_item/job_specific/holyarmor
 	name = "Holy Armor"
 	desc = "A set of blessed Holy Armor that shoots out lightning whenever you're attacked."
 	item = /obj/item/clothing/suit/armor/riot/knight/templar/holy
-	cost = 15
+	cost = 10
 	jobs = list("Chaplain")
 
 //BARTENDER
 /datum/uplink_item/job_specific/shotgunammo
 	name = "Shotgun ammo kit"
 	desc = "An assortment of various syndicate approved shotgun shells."
 	item = /obj/item/weapon/storage/box/syndie_kit/shotguntender
-	cost = 5
+	cost = 2
 	jobs = list("Bartender")
 
 //CLOWN + MIME
 /datum/uplink_item/job_specific/caneshotgun
 	name = "Cane Shotgun + Assassination Darts"
 	desc = "A specialized, one shell shotgun with a built-in cloaking device to mimic a cane. The shotgun is capable of hiding it's contents and the pin alongside being supressed. Comes with 6 special darts and a preloaded shrapnel round."
 	item = /obj/item/weapon/storage/box/syndie_kit/caneshotgun
-	cost = 15
+	cost = 10
 	jobs = list("Clown","Mime")
 //CLOWN
 /datum/uplink_item/job_specific/honkrender
 	name = "Honk Render"
 	desc = "A blade capable of tearing apart the fabric of reality and summoning several clowns from the Clown Planet itself. They're mostly passive, until someone hits one of them, after which they enter a frenzy, attacking any and all in a collective group effort. The blade itself is also useful as a slashing weapon."
 	item = /obj/item/weapon/veilrender/honkrender
-	cost = 7
+	cost = 6
```

Comment:
They're not really effective and they only attack when attacked. They're hilarious but they also make you kill on sight as the clown as you're the only one who could do it. I think it's fine